examples/alice_http/lib/main.dart

```diff
@@ -16,11 +17,8 @@ class MyApp extends StatefulWidget {
 class _MyAppState extends State<MyApp> {
   late final AliceHttpAdapter _aliceHttpAdapter = AliceHttpAdapter();
 
-  late final Alice _alice = Alice(
-    showNotification: true,
-    showInspectorOnShake: true,
-    maxCallsCount: 1000,
-  )..addAdapter(_aliceHttpAdapter);
+  late final Alice _alice = Alice(configuration: AliceConfiguration())
```

Comment:
Since this is an example, declare the configuration in a separate variable, just so that it's clearer for a new user.